chrome/content/prefs.js

```diff
@@ -45,6 +48,31 @@ var otrPref = {
     }
   },
 
+  coniksTabInit: function() {
+    document.getElementById("coniksTab").hidden = false;
+    let accountList = document.getElementById("coniksaccountlist");
```

Comment:
Should we refactor this into a helper that takes `(listElement, func)`, and then we're pass `("coniksaccountlist", this.displayAcountPolicy)` and above `("accountlist", this.swapFinger)`?